packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -410,6 +412,17 @@ export interface ISummaryRuntimeOptions {
     summarizerOptions?: Readonly<Partial<ISummarizerOptions>>;
 }
 
+/**
+ * Options for op compression.
+ * @experimental - Not ready for use
+ */
+export interface ICompressionRuntimeOptions {
+    /**
+     * The minimum size the content payload must exceed before it is compressed. Undefined if compression is disabled.
```

Comment:
Maybe other way around - "compression is disabled if undefined"

Reply:
Having two undefined creates some challenges. Maybe we should say that if compressionOptions is undefined, then compression is disabled. And make this required field?

Reply:
I would rather do it that way, but unfortunately `ICompressionRuntimeOptions` can't be undefined because `ContainerRuntime`'s constructor wraps the runtime options in `Readonly<Required<IContainerRuntimeOptions>>`. I did update the comment though.